acceptance/openstack/dns/v2/quotas.go

```diff
@@ -0,0 +1,25 @@
+//go:build acceptance
+// +build acceptance
+
+package v2
+
+import (
+	"testing"
+
+	"github.com/gophercloud/gophercloud/acceptance/clients"
+	"github.com/gophercloud/gophercloud/acceptance/tools"
+	"github.com/gophercloud/gophercloud/openstack/dns/v2/quotas"
+	th "github.com/gophercloud/gophercloud/testhelper"
+)
+
+func TestSchedulerStatsList(t *testing.T) {
+	client, err := clients.NewDNSV2Client()
+	th.AssertNoErr(t, err)
+
+	randomUUID := "513788b0-4f1b-4107-aee2-fbcdca9b9833"
```

Comment:
This is likely going to generate an error, querying for a project that doesn't exist.

I'd expect we validate both Get and Update() functions.

You could for instance request the current quota, modify a value, then check it was effectively modified.
Then, if you implement quota reset, you can restore the original quota and verify it was changed back to the initial value.

You can use `clients.RequireAdmin(t)` to request admin creds if needed.